src/components/index.ts

```diff
@@ -1,31 +1,31 @@
+import Alert from './Alert/Alert';
```

Comment:
We kind of lost touch on this file; quite a few things weren't correct. Might need to update our process to make sure it stays current.